src/service/HTTPService.tsx

```diff
@@ -198,4 +201,18 @@ export const HttpService = {
       config
     );
   },
+  getDataAgreement: async (): Promise<PurposeForDataProvider[]> => {
```

Comment:
Rename this to `listDataAgreements`, as this is used for listing all data agreements.